### PR TITLE
Add changeset for PR #5739

### PR DESCRIPTION
**Pull request by hsubox76**
Changeset for https://github.com/firebase/firebase-js-sdk/pull/5739

### CODE REVIEW

File: .changeset/yellow-suns-clean.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/auth': patch
+---
+
+Prevent React Native logging a warning about deprecation of `AsyncStorage` if user has provided the non-deprecated version.
```
Comment (egilmorez) [inline .changeset/yellow-suns-clean.md:5]: The preposition could clarify a little: "Prevent React Native from logging..."

Comment (egilmorez) [inline .changeset/yellow-suns-clean.md:5]: Is "user" definitely the right term?  Or could it be "the developer" or even "the package.json file" or wherever this is defined?

I clearly do not know :)  But if we go with this, I'd suggest "if the user"

**Review by egilmorez — COMMENTED** [review]
Couple of nits for you to look at, thanks!

File: .changeset/yellow-suns-clean.md (hunk 1 above)
Comment (hsubox76) [reply .changeset/yellow-suns-clean.md:5]: "developer" makes sense.

Comment (hsubox76) [reply .changeset/yellow-suns-clean.md]: Done.

**Pull request merged**